en-US/editor/editor_functions.md

```diff
@@ -32,30 +31,37 @@ an upload).
 * **Move Objects To Layer** - Moves a selected set of notes to a specific layer. **(requires notes to be selected with the select tool)**
 * **Edit Metadata** - Lets you edit the metadata of the current song.
 * **Edit Timing Points** - Lets you add, remove, or modify timing points which can help time the song.
 * **Edit Scroll Velocities** - Lets you modify the map with scroll velocities that change the speed of how the song scrolls.
 * **Set Song Select Preview Time** - Sets the location (at the current point) in which the song select will start the song preview at.
 * **Apply Offset to Map** - Applies an offset to the entire map, which shifts timing points, scroll velocities, and notes by a given ms.
-* **Apply Modifier to Map** - Applies a note modifier to your map so you can see how it would look.
+* **Resnap All Notes** - Options to resnap all notes.
+  * **Resnap to currently selected snap** - Resnaps all notes to the currently selected beat snap, which is shown in the brackets.
+  * **Resnap to 1/16 and 1/12 snaps** - Resnaps all notes to 1/16 and 1/12 snaps. Notes will snap to the closest beat snap divisor.
+* **Resnap Selected Notes** - Options to resnap notes highlighted with the "Select" tool.
+  * **Resnap to currently selected snap** - Resnaps selected notes to the currently selected beat snap, which is shown in the brackets.
+  * **Resnap to 1/16 and 1/12 snaps** - Resnaps selected notes to 1/16 and 1/12 snaps. Notes will snap to the closest beat snap divisor.
+* **Apply Modifier to Map** - Applies a note modifier to your map, such as Mirror, No Long Notes, Full Long Notes, or Inverse. This action is irreversible. 
 
 ## View
 
 * **Display Gameplay Preview** - Shows a little gameplay preview at the side of your editor field so you can see how the song would look as you play it. You can also
 press [Tab] during this to take control and start playing yourself.
 * **Reference Difficulty** - Displays another difficulty of the same mapset side by side so you can see how the two difficulties differ.
 * **Background Brightness** - Changes how bright or dark the background image is.
-* **Beat Snap Divisor** - Lets you set the beat snap of the given song (you can also adjust this by holding CTRL and using your mouse wheel)
+* **Beat Snap Divisor** - Lets you set the beat snap of the given song. You can also adjust this by holding CTRL and using your mouse wheel.
 * **Beat Snap Color** - Changes the color of the beat snaps to one of three color schemes.
-* **Scale Scroll Speed** - Scales the speed of the editor field to BPM changes (faster BPM = faster scroll speed)
-* **Place Objects On Nearest Tick** - Places objects on the nearest beat tick from your mouse position
-* **Show Waveform** - Shows the blue waveform on the editor field.
+* **Scale Scroll Speed** - Enable/Disable scaling the speed of the editor field to BPM changes. (faster BPM = faster scroll speed)
```

Comment:
Not directly relevant to the PR, but this description is wrong. It refers to the scaling of the playback rate, BPM has nothing to do with it.